src/components/SearchBar/SearchBar.test.js

```diff
@@ -12,12 +12,15 @@ describe('SearchBar', () => {
     expect(inputElement).toBeInTheDocument();
   });
 
-  it('changes text when typed into', () => {
+  it('erases typed text on key down', () => {
     const typedText = 'abcde';
 
     const inputElement = screen.getByPlaceholderText(/search/i);
```

Comment:
I would recommend to use a [test-id](https://testing-library.com/docs/queries/bytestid/) for this case, because maybe the placeholder could change but the component will always be the same